website/cue/reference/components/sources/host_metrics.cue

```diff
@@ -116,6 +117,72 @@ components: sources: host_metrics: {
 				unit:    "seconds"
 			}
 		}
+		cgroups: {
+			common:      false
+			description: #"Options for the "cgroups" metrics collector. This collector is only available on Linux systems, as the cgroups (control groups) system is only present on Linux."#
+			required:    false
+			type: object: options: {
+				base: {
+					common:      false
+					required:    false
+					description: "The base cgroup name to provide metrics for"
```

Comment:
It is possible, but I went for the simple case first.

Reply:
Sure, I guess I'm just wondering how likely it is that users will have multiple cgroups they'll want to track. If we anticipate changing it soon, it'd probably be worth just doing it now to avoid the deprecation dance.

Reply:
I have no good notion if users will want to report multiple cgroups beyond the hierarchy. Doing it will take some work, as there is the potential for one named group to be inside another named group, and we wouldn't want to report the metrics multiple times. On the other hand, recursing the entire groups tree just to find the named groups is wasted effort, so filtering could be tricky.